common/src/main/java/com/microsoft/identity/common/internal/broker/MicrosoftAuthClient.java

```diff
@@ -106,6 +111,19 @@ private Intent getIntentForAuthService(final Context context) {
         return authServiceToBind;
     }
 
+    private boolean isMicrosoftAuthServiceSupported(@NonNull final Context context, @NonNull final String packageName) {
+        if (packageName == null || packageName.length() == 0) {
```

Comment:
We should remove the defensive code here.... there's no such thing as an `""` or `null` package name so we shouldn't even suggest it's possible